roadmap/implementers-guide/src/runtime/disputes2.md

```diff
@@ -0,0 +1,54 @@
+# Disputes
+
+## Motivation
+
+If the transaction can be verified the defined verification code, it is good. If not, there is a dispute
+about a block, which made it through the backing stage, but was then during secondary checking found
+to be invalid.
+At this point the validator set can not be trusted (since they voted for the block despite something being
+fishy at the very least) or the validator that blew the whistle has ulterior motives to do so.
+In either way, there are malicious validators around.
+As a consequence, all validators at the time of block backing, are being notified via broadcast of the first challenging vote.
+Validators that backed the candidate implicitly voted for backing the candidate and may not vote a second time.
+That would be equal to an attempted double vote and would be slashed accordingly.
+
+## Initiation
+
+A dispute is initiated by one secondary checker creating and gossiping a vote, that challenges the vote.
+Secondary checkers run the validation code and use this to verify the transactions.
+
+After a secondary checker challenged a block, all validators that received the gossiped vote, check the
+block themselves via the validation code.
```

Comment:
"check the block" could "reconstruct and check the block" but maybe that's implicit based upon everything else